content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Developer Summit 2"
+date: 2024-09-19
+draft: false
+description: "Summary of the 2024 Scientific Python Developer summit."
+tags: ["Summit", "Scientific-Python"]
+displayInList: true
+author:
+  [
+    "Stéfan van der Walt",
+    "Jarrod Millman",
+    "Brigitta Sipőcz",
+    "Pamphile Roy",
+    "Matt Haberland",
+    "Elliott Sales de Andrade",
+    "Matthew Feickert",
+    "Melissa Mendonça",
+  ]
+
+resources:
+  - name: featuredImage
+    src: "group.jpg"
+    params:
+      description: "Group picture of (most of the) summit attendees."
+      showOnTop: true
+
+summary: |
+  The 2024 Scientific Python Developer summit was held 3–5 June in Seattle. Here's a summary of what we did.
+---
+
+This post is a while overdue, but it's been a busy summer for everyone!
+
+In June, several of us got together for the "annual" (well, we've had it twice now) Scientific Python developer summit in Seattle.
+Our friends at the eScience Institute were again kind enough to host us.
+This time around, we made the event a bit shorter: a week is a long time to take off, and it also gets a bit overstimulating/exhausting, given the amount of conversation and work that takes place.
```

Comment:
This sentence makes me hesitate a bit; I *liked* the full week format of the first one, and found 3 days felt a bit short. I worry that saying "5 days was exhausting" might make it harder to get funding for a full week if we decided to go back to that format. 

(Also I thought part of the complication was overlapping with commencement --- not that I think we should mention that, but it feels a bit disingenuous to only say "we shortened because 5 days was too long"). Would it be OK to just say "this time was shorter" without explanation, or with a vague explanation like "for logistical reasons"?

```suggestion
This time around, we made the event a bit shorter (for logistical reasons).
```

Reply:
I noted that we shortened it to avoid clashing with graduation.